index.js

```diff
@@ -4,140 +4,219 @@ window.addEventListener("load", () => {
   const eqButton = document.querySelector(".eButton");
   const numDisplay = document.querySelector(".numDisplay");
   const opArray = ["add", "subtract", "multiply", "divide"];
-  let currentString = numDisplay.textContent;
+  let currentString = "";
+  //This should always have STRING data. numDisplay, despite the name, should also always have STRING data.
   let firstNum = "";
+  //This should always have NUMBER data, OR a BLANK STRING (bad practice - figure out a better solution later).
```

Comment:
You're not far off here. Normally you'd use `null` or `undefined` to put a variable in a "not set" state.

Reply:
Noted. Will have to refresh myself on null/undefined checks for next project.

Reply:
It's just `firstNum != null` to check for null or undefined, or `firstNum !== null` / `firstNum !== undefined` to check for either one specifically.